raspberry/bzzz/sensors/gnss.py

```diff
@@ -131,6 +134,25 @@ def __get_measurements_in_background_t(self, serial_path, baud):
         ser.close()
         return
 
+    def GNSS_altitude_Initilisation(self):
```

Comment:
1. Follow PEP8 (`gnss_altitude_initilisation`)
2. Does this need to be a public method?

Reply:
@alphaville when using the initialisation value, we will do the subtraction outside the gnss class (In main presume) when getting input data for the Kalman Filter? so I needs to be public

Reply:
@pdavid747 You could put `gnss_altitude_initilisation` in the constructor